text/0098-metric-instruments-explained.md

```diff
@@ -0,0 +1,207 @@
+  - if asynchronous use `ValueObserver` to record a single measurement nearing the end of a collection interval.
+
+### Sync vs Async instruments
+
+Synchronous instruments are called in a request context, meaning they potentially have an associated tracing context and distributed correlation values.  Multiple metric events may occur for a synchronous instrument within a given collection interval.
+
+Asynchronous instruments are reported by a callback, once per collection interval, and lack request context.  They are permitted to report only one value per distinct label set per period.  If the application observes multiple values in a single callback, for one collection interval, the last value "wins".
+
+### Temporal quality
+
+Measurements can be described in terms of their relationship with time.
+
+Delta measurements are those that measure a change to a sum.  Delta instruments are usually selected because the program does not need to compute the sum for itself, but is able to measure the change.  In these cases, it would require extra state for the user to report cumulative values and reporting deltas is natural.
+
+Cumulative measurements are those that report the current value of a sum.  Cumulative instruments are usually selected because the program maintains a sum for its own purposes, or because changes in the sum are not instrumented.  In these cases, it would require extra state for the user to report delta values and reporting cumulative values is natural.
+
+Instantaneous measurements are those that report a non-additive measurement, one where it is not natural to compute a sum.  Instantaneous instruments are usually chosen when the distribution of values is of interest, not only the sum.
+
+### Function names
+
+Synchronous delta instruments support an `Add()` function, signifying that they add to a sum and are not cumulative.
+
+Synchronous instantaneous instruments support a `Record()` function, signifying that they capture individual events, not only a sum.
+
+Asynchronous instruments all support an `Observe()` function, signifying that they capture only one value per measurement interval.
+
+### Rate support
+
+Rate aggregation is supported for Counter and SumObserver instruments in the default implementation.
+
+The `UpDown-` forms of additive instrument are not suitable for aggregating rates because the up- and down-changes in state may cancel each other. 
+
+Non-additive instruments can be used to derive a sum, meaning rate aggregation is possible when the values are non-negative. There is not a standard non-additive instrument with a non-negative refinement in the standard.
+
+### Defalt Aggregations
+
+Additive instruments use `Sum` aggregation by default, since by definition they are used when only the sum is of interest.
+
+Instantaneous instruments use `MinMaxSumCount` aggregation by default, which is an inexpensive way to summarize a distribution of values.
```

Comment:
Was [this](https://github.com/open-telemetry/opentelemetry-specification/issues/462) spec issue about deciding whether MinMaxSumCount or just SumCount aggregation is required for measure instruments? If yes, what section of this otep #98 help resolve that?

Reply:
It was, but it wasn't clear. Here I've added a specific answer: https://github.com/open-telemetry/oteps/pull/98/commits/a3af52ba0f1dc4a2ff19f7629710d00291570edd